pages/docs/installation/installation-source-dependencies.md

```diff
@@ -151,21 +151,13 @@ The following header-only Boost libraries are also needed: 'asio', `vmd`, `geome
    The directory you chose as prefix now contains libraries in `<prefix>/lib` and the all the Boost headers in `<prefix>/include`.
    You may now safely remove the boost directory from step 1.
 
-4. If you selected `/usr/local` as prefix, update the dynamic linker's run-time bindings:
-
+4. If you selected `/usr/local` as prefix, you do not have to do anything else. If you used a custom `<prefix>` you have to add the following line to your `.bashrc`:
```

Comment:
This depends on the system. Ubuntu ignores `/usr/local` by default. Maybe you have added it to your respective paths? Or is it just that newer Boost changed something?

Reply:
I was working on CoolMUC and building Boost from source. Maybe the point with *"...update the dynamic linker's run-time bindings..."* is still relevant if you use `/usr/local` as a prefix (On Ubuntu I use the system Boost). We can also revert this bit and only keep the part about `CMAKE_PREFIX_PATH`.